km_api/know_me/models.py

```diff
@@ -266,6 +267,43 @@ def get_absolute_url(self):
         """
         return reverse('know-me:km-user-detail', kwargs={'pk': self.pk})
 
+    def get_emergency_contact_detail_url(self, request=None):
+        """
+        Get the absolute URL of the instance's emergency contact view.
+        Args:
+            request (optional):
+                A request used as context when constructing the URL. If
+                given, the resulting URL will be a full URI with a
+                protocol and domain name.
+        Returns:
+            str:
+                The absolute URL of the instance's gallery view.
+        """
+        return reverse(
+            'know-me:emergency-contact-detail',
+            kwargs={'pk': self.pk},
+            request=request)
+
+    def get_emergency_contact_list_url(self, request=None):
```

Comment:
I believe adding a test for this method would improve your code coverage. Also, tests are good.